### PR TITLE
GAWB-1340: visual tweaks for Library search results

### PR DESCRIPTION
**Pull request by davidangb**
* decrease font size of "Search Filters:" and "Search Results:" headers to match mockups in jira ticket
* increase spacing between main header and "Search Filters:"/"Search Results:" headers to match mockups and to match spacing on workspace summary page
* increase spacing between "Search Filters:"/"Search Results:" headers and main page body
* tighten up table: smaller column widths, less row spacing, slight decrease of font size (14.4px->14px), but add a margin between header and rows
* move some CSS styles from the table's `:cell-content-style` to `:body-style` so we're outputting less HTML and we're more efficient in the DOM

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs [hunk 1]
```diff
@@ -33,7 +33,7 @@
        :toolbar
        (fn [{:keys [reorderer]}]
          [:div {:style {:display "flex" :alignItems "top"}}
-          [:div {:style {:fontWeight 700 :fontSize "125%"}} "Search Results: "
+          [:div {:style {:fontWeight 700 :fontSize 20 :marginBottom "1em"}} "Search Results: "
```
Comment (zarsky-broad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs]: Absolute sizes always make me a bit squidgy, so consider representing this as 125%. Not the end of the world, though.

File: src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs [hunk 2]
```diff
@@ -42,22 +42,21 @@
                    " found"))]]
           flex-strut
           reorderer])
-       :body-style {:fontSize nil :fontWeight nil}
-       :row-style {:backgroundColor nil :height 22}
-       :cell-content-style {:padding nil
-                            :color (:text-light style/colors)
-                            :fontSize "90%"}
-       :columns [{:header "Dataset Name" :starting-width 300
+       :body-style {:fontSize 14 :fontWeight nil :marginTop 4
```
Comment (zarsky-broad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs]: See above re: font size

File: src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs [hunk 3]
```diff
@@ -107,7 +106,7 @@
   {:render
    (fn [{:keys [props]}]
      [:div {}
-      [:div {:style {:fontWeight 700 :fontSize "125%"}} "Search Filters:"]
+      [:div {:style {:fontWeight 700 :fontSize 20 :marginBottom "1em"}} "Search Filters:"]
```
Comment (zarsky-broad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs]: See above re: font size

**Review by zarsky-broad — APPROVED** [review]

**Comment by davidangb** [comment]
now with relative font sizes. The 20px fonts were previously at 125%, so I just reverted that change.

**Pull request merged**